monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py

```diff
@@ -20,6 +20,10 @@ def __init__(self):
 
     def run(self):
         results = [pba.run() for pba in self.modify_shell_startup_PBA_list()]
+        if not results:
+            results = [
+                ("Modify shell startup files PBA failed: Unable to find any regular users.", False)
```

Comment:
Are we sure this is the only case when `results` is empty? Perhaps something like "PBA failed, most likely since Monkey was unable to find any regular users" would be safer.

Reply:
We know that it is not the only case. New issue is for that #1509 .

Reply:
👍 
For now, I'd still suggest we reword this.

Reply:
In the case of #1509, there will still be results. The only case we're aware of at the moment that causes this is that this PBA failed to find any regular users